### PR TITLE
Account, Metadata, Namespace And Receipts

### PR DESCRIPTION
**Pull request by fboucquez**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Once your changes and tests are ready to submit for review:
     
 Then sit back and wait. There will probably be discussion about the pull request and, if any changes are needed, we would love to work with you to get your pull request merged into symbol-sdk-java.
 
-*CONTRIBUTING.md is based on [CONTRIBUTING-template.md](https://github.com/nayafia/contributing-template/blob/master/CONTRIBUTING-template.md)* , [elasticsearch/CONTRIBUTING](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md) and [spark/CONTRIBUTING](https://github.com/apache/spark/blob/master/CONTRIBUTING.md)
+*CONTRIBUTING.md is based on [CONTRIBUTING-template.md](https://github.com/nayafia/contributing-template/blob/main/CONTRIBUTING-template.md)* , [elasticsearch/CONTRIBUTING](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md) and [spark/CONTRIBUTING](https://github.com/apache/spark/blob/main/CONTRIBUTING.md)
```
Comment (rg911) [inline CONTRIBUTING.md:92]: only our repos changed from master to main

Comment (fboucquez) [reply CONTRIBUTING.md:92]: It seems like github automatically redirects main to master if main is not there. Should we remove _master_ from our source code even when it's not ours as possible?

Comment (rg911) [reply CONTRIBUTING.md:92]: Not sure about that. We renamed our master branch to main via an internal vote, but that doesn't mean external repo will / should follow

Comment (fboucquez) [reply CONTRIBUTING.md]: ok, back to master...

**Pull request merged**